contents/docs/surveys/creating-surveys.mdx

```diff
@@ -119,7 +119,7 @@ You can display your survey to specific users based on:
 
 ### Completion conditions
 
-This enables you to stop your survey once you receive a specific number of responses. Note that because it takes time for PostHog to process responses, this is a rough guideline. Your survey might receive slightly more responses than you set.
+This enables you to stop your survey once you receive a specific number of responses. Note that because it takes time for PostHog to process responses, this is a rough guideline. Your survey might receive slightly more responses than you set. The completion conditions properties are evaluated with internal feature flags.
```

Comment:
$survey_responded/x
$survey_dismissed/x
$survey_response
$survey_response_x
those are the props I believe are evaluated, TBC

Reply:
https://github.com/PostHog/posthog/blob/39940ea5095e8344c1934793c4b6fe138f92bc90/posthog/api/survey.py#L588-L594